MIGRATING.MD

```diff
@@ -64,32 +79,37 @@ OpenTelemetry Java Instrumentation.
 ### Configuration setting changes
 
 These SignalFx Java Agent system properties correspond to the following
-OpenTelemetry system properties:
+OpenTelemetry system properties (NOTE: for exporter - specific properties we use `jaeger` as an example):
 
 | SignalFx system property | OpenTelemetry system property |
 | ------------------------ | ----------------------------- |
-| `signalfx.service.name` | `otel.resource.attributes=service.name=my-java-app` |
+| `signalfx.service.name` | `otel.resource.attributes=service.name=<name of the service>` |
+| `signalfx.env` | `otel.resource.attributes=deployment.environment=<name of the environment>` |
 | `signalfx.endpoint.url` | `otel.exporter.jaeger.endpoint` |
-| `signalfx.tracing.enabled` | `otel.trace.enabled` |
-| `signalfx.span.tags` | `otel.resource.attributes` |
-| `signalfx.recorded.value.max.length` | `otel.config.max.attr.length` |
-| `signalfx.db.statement.max.length` | `otel.config.max.attr.length` | 
+| `signalfx.tracing.enabled` | `otel.javaagent.enabled` |
+| `signalfx.integration.<name>.enabled=false` | `otel.instrumentation.[id].enabled=false` | 
```

Comment:
Should we keep the template <name> the same on both sides?

```suggestion
| `signalfx.integration.<name>.enabled=false` | `otel.instrumentation.<name>.enabled=false` | 
```

Reply:
In OTEL instrumentation it's called "instrumentation ID" so let's stay coherent here.